### PR TITLE
43 Add person base class

### LINKED ISSUE #43: Make a `Person` base class

### PR DESCRIPTION
**Pull request by guangxuan**
Closes #43
Add Person as a Base class
Add PersonBuilder and TypicalPersons for convenience of testing in the future

### CODE REVIEW

File: src/test/resources/view/UiPartTest/validFileWithFxRoot.fxml [hunk 1]
```diff
@@ -1,6 +1,7 @@
 <?xml version="1.0" encoding="UTF-8"?>
 
+<?import seedu.address.ui.TestFxmlObject?>
 <fx:root type="seedu.address.ui.TestFxmlObject" xmlns="http://javafx.com/javafx"
-            xmlns:fx="http://javafx.com/fxml">
+         xmlns:fx="http://javafx.com/fxml">
```
Comment (waynetee) [inline src/test/resources/view/UiPartTest/validFileWithFxRoot.fxml:5]: Was this intended?

**Review by waynetee — APPROVED** [review]
LGTM

**Review by lizchow — APPROVED** [review]
LGTM!

File: src/test/resources/view/UiPartTest/validFileWithFxRoot.fxml (hunk 1 above)
Comment (guangxuan) [reply src/test/resources/view/UiPartTest/validFileWithFxRoot.fxml]: No. Reverted.

**Pull request merged**